astropy/units/quantity.py

```diff
@@ -607,6 +618,15 @@ def to(self, unit, equivalencies=[]):
         return self._new_view(new_val, unit)
 
     @property
+    def info(self):
```

Comment:
@embray earlier suggested to use `lazyproperty` here, so that this is shorter. I think that makes sense:
```
@lazyproperty
def info(self):
    return QuantityColumnInfo(self)
```